### PR TITLE
feat(virtualbursting): add limits to the configuration table

### PR DESCRIPTION
**Pull request by Poltergeist**
This adds the virtual bursting limit settings to the configuration table.

CLOSES DCOS-62250

### CODE REVIEW

File: src/js/__tests__/__snapshots__/tslint-test.ts.snap [hunk 1]
```diff
@@ -523,6 +523,7 @@ exports[`tslint snapshot should be up to date 1`] = `
 /plugins/services/src/js/service-configuration/PodNetworkConfigSection.tsx - Shadowed name: 'endpoints'
 /plugins/services/src/js/service-configuration/PodNetworkConfigSection.tsx - Shadowed name: 'memo'
 /plugins/services/src/js/service-configuration/PodStorageConfigSection.tsx - Use a conditional expression instead of assigning to 'type' in multiple places.
+/plugins/services/src/js/service-configuration/ServiceGeneralConfigSection.tsx - == should be ===
```
Comment (pierrebeitz) [inline src/js/__tests__/__snapshots__/tslint-test.ts.snap]: let's please rather reduce the overall number of lints in every PR.

File: src/js/__tests__/__snapshots__/typecheck-test.ts.snap [hunk 2]
```diff
@@ -4200,6 +4200,8 @@ plugins/services/src/js/service-configuration/PodContainerArtifactsConfigSection
 plugins/services/src/js/service-configuration/PodContainerArtifactsConfigSection.tsx: error TS2322: type * is not assignable to type *.
 plugins/services/src/js/service-configuration/PodContainerConfigSection.tsx: error TS2322: type * is not assignable to type *.
 plugins/services/src/js/service-configuration/PodContainerConfigSection.tsx: error TS2322: type * is not assignable to type *.
+plugins/services/src/js/service-configuration/PodContainerConfigSection.tsx: error TS2322: type * is not assignable to type *.
+plugins/services/src/js/service-configuration/PodContainerConfigSection.tsx: error TS2322: type * is not assignable to type *.
```
Comment (pierrebeitz) [inline src/js/__tests__/__snapshots__/typecheck-test.ts.snap]: let's please rather reduce the overall number of type errors in every PR.

File: tests/pages/services/ServiceDetail-cy.js [hunk 3]
```diff
@@ -267,7 +267,7 @@ describe("Service Detail Page", () => {
       });
     });
 
-    it("edit config button opens the edit flow", () => {
+    it("edit config button ests/pages/services/ServiceDetail-cy.js.origopens the edit flow", () => {
```
Comment (pierrebeitz) [inline tests/pages/services/ServiceDetail-cy.js]: interesting name

Comment (Poltergeist) [reply tests/pages/services/ServiceDetail-cy.js]: 😱

File: src/js/__tests__/__snapshots__/tslint-test.ts.snap (hunk 1 above)
Comment (Poltergeist) [reply src/js/__tests__/__snapshots__/tslint-test.ts.snap]: ✅

File: src/js/components/ConfigurationMapHeading.tsx [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
-import PropTypes from "prop-types";
 import * as React from "react";
 
 import DetailViewSectionHeading from "./DetailViewSectionHeading";
 
-const ConfigurationMapHeading = props => {
-  const { children, className, level } = props;
+const ConfigurationMapHeading = (props: {
+  children?: React.ReactNode;
+  className?: string | string[] | object;
```
Comment (pierrebeitz) [inline src/js/components/ConfigurationMapHeading.tsx]: you're looking for 
```
import { ClassValue } from "classnames/types";
  ...
  className?: ClassValue;
  ...
```

**Pull request merged**